compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -10858,15 +10871,12 @@ OMR::Z::TreeEvaluator::arraycmpHelper(TR::Node *node,
             if (isFoldedIf)
                {
                if (isIfxcmpBrCondContainEqual)
-                  setStartInternalControlFlow(cg, node, generateS390BranchInstruction(cg, TR::InstOpCode::BRC, TR::InstOpCode::COND_BL, node, compareTarget),
-                                              isStartInternalControlFlowSet);
```

Comment:
control flow always start before short circuit branch. This is no longer necessary.